internal/scorecard/alpha/scorecard.go

```diff
@@ -100,12 +106,22 @@ func selectTests(selector labels.Selector, tests []Test) []Test {
 }
 
 // runTest executes a single test
-func (o Scorecard) runTest(test Test) (result *v1.Pod, err error) {
+func (o Scorecard) runTest(ctx context.Context, test Test) (result v1alpha2.ScorecardTestResult, err error) {
```

Comment:
This function looks like it either returns an empty result with an error or a valid result with no error, so it would me more idiomatic to make the result a pointer.
```suggestion
func (o Scorecard) runTest(ctx context.Context, test Test) (result *v1alpha2.ScorecardTestResult, err error) {
```